lms/templates/support/program_enrollments_inspector.html

```diff
@@ -21,7 +21,7 @@
 
 <%block name="content">
 	<section class="container outside-app">
-        <h3> Program Enrollments Inspector </h3>
+        <h1> Program Enrollments Inspector </h1>
```

Comment:
There's a ticket to spruce up the UI, right? Can you drop a `# TODO: Improve this UI (MST-XX)`-esque comment somewhere in this PR?